site/about.html

```diff
@@ -1,28 +1,42 @@
+<!doctype html>
+
 <html>
 	<head>
 		<title>About me</title>
-		<link rel="stylesheet" href="style.css" >
+
+		<link rel="stylesheet" href="style.css">
+		<link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Lato:400,900' rel='stylesheet' type='text/css'>
 	</head>
 
 	<body>
-		<nav>
-			<a href="about.html">About Me</a>
-			<a href="portfolio.html">Portfolio</a>
-		</nav>
+		<ul class="top_nav">
+			<li class="top_nav_item"><a href="about.html" class="active">About me</a></li>
+			<li class="top_nav_item"><a href="portfolio.html" class="li a">Portfolio</a></li>
```

Comment:
I do not think the classes on the ```<a>``` tag are necessary here.  I do not see them being used in the CSS file and if you need to style these links, you can always set the selector as:

li a {} 

^^ Notice, there are no dots before the li or a above

-- or --

.top_nav_item a

^^ Using the class name from the ```<li>``` tag.